worlds/animal-well/regions.py

```diff
@@ -483,7 +483,7 @@ class AWData(NamedTuple):
         lname.candle_dog_dark:
             AWData(AWType.location, [[iname.match]]),  # todo: figure out what we're doing for matches/candles
         dog_chinchilla_skull:
-            AWData(AWType.region, [[iname.bubble]]),  # can get here without bubble jumps
+            AWData(AWType.region, [[iname.bubble], [iname.remote], [iname.disc], [iname.ball], [iname.top, iname.weird_skips]]),  # hit a switch with any number of things, or bubble up there yourself
```

Comment:
For the ball here, what's the intent? I'm trying this and it doesn't feel super intuitive, just kinda randomly throwing.
Mostly, I'm asking since you have weird_skips down for the top, and this feels way easier with the top